src/ev3_bluetooth.h

```diff
@@ -0,0 +1,62 @@
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef ev3_bluetooth_h
+#define ev3_bluetooth_h
+
+#define BLUETOOTH_ADDRESS_LENGTH            sizeof("00:00:00:00:00:00")
+#define MAX_BLUETOOTH_NAME_LENGTH           128
+
+void BluetoothInit();
+
+typedef int BluetoothConnectionHandle;
+
+/**
+ * Connect to the remote device given his bluetooth name or address.
+ * If the connection can't be established, a new attempt will be tried after 1
+ * second.
+ * @param nameOrAddress
+ * @param attempts maximum number of attempts. This argument should be >= 1,
+ *        otherwise no attempt will be tried (and the connection will never be
+ *        established)
+ * @return Connection handle or -1 if wasn't possible to find the address given the device name
+ */
+BluetoothConnectionHandle ConnectTo(const char * nameOrAddress, int attempts);
+
+/**
+ * Wait for an incoming bluetooth connection from another device
+ * @return Connection handle or -1 if there was an error while waiting for the connection
+ */
+BluetoothConnectionHandle WaitConnection();
```

Comment:
`(void)` here and everywhere else. There's no reason to use `()` except maybe for function pointers.

Reply:
69ea7a0